pyproject.toml

```diff
@@ -16,13 +16,15 @@ classifiers = [
     "Framework :: Apache Airflow :: Provider",
 ]
 dependencies = [
-    "apache-airflow",
-    "pytest",
-    "requests",
-    "responses",
-    "setuptools"
+    "apache-airflow>=2.10.0,<=2.10.5",
```

Comment:
@RanjeetMishra why we need `<=2.10.5`

Reply:
I see this is the last version in 2.x.x. makes sense.

Reply:
Yes 3.x version need some changes.. we can wait for some time for this. Without upper limit, it picks up latest version and some tests are failing.